src/main/java/org/openlmis/requisition/domain/requisition/StatusChangeValidationService.java

```diff
@@ -64,8 +64,13 @@ public StatusChangeValidationService(Requisition requisition, LocalDate currentD
     validators.add(new RequestedQuantityValidator(requisition, orderables));
     validators.add(new StockAdjustmentsValidator(requisition, orderables));
     validators.add(new TotalFieldValidator(requisition, requisition.getTemplate(), orderables));
-    validators.add(new TotalReceivedQuantityValidator(requisition,
-        requisition.getTemplate(), orderables));
+
+    boolean skipTotalReceivedQuantityValidation = Boolean.TRUE.equals(requisition.getTemplate()
```

Comment:
Once you change the `getPatientsTabEnabled` to boolean, the `Boolean.TRUE.equals` won't be needed.

Reply:
@pwargulak good point.
The same could be refactored here: https://github.com/OpenLMIS/openlmis-requisition/blob/397cd4c215dbaa865507280dc2b3762b64f9f5a9/src/main/java/org/openlmis/requisition/validate/RequisitionTemplateDtoValidator.java#L108